plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java

```diff
@@ -365,7 +359,7 @@ private static RangeBoundValue buildRangePartitionBound(KuduTable table, byte[]
             Schema schema = table.getSchema();
             PartitionSchema partitionSchema = table.getPartitionSchema();
             PartitionSchema.RangeSchema rangeSchema = partitionSchema.getRangeSchema();
-            List<Integer> rangeColumns = rangeSchema.getColumns();
+            List<Integer> rangeColumns = rangeSchema.getColumnIds();
```

Comment:
This can also be a separate commit as `getColumns` is a deprecated API

Reply:
Updated

Reply:
Can we add it as first commit ? Can we rename the commit message as `Remove deprecated API usage in KuduTableProperties`